### DISCUSSION TITLE
How do we represent structures within complexes?

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/QRiS#273] [Ideas & Feature Requests]
We have two use cases:
1. Design structures (by points or crests) and draw complex polygon around them (a lasso), and sum up structures drawn, and allow +/- specification at complex (not structure level)
2. Draw complex polygon, and specify how many structures (just by 3 types and other) and +/- specification.

How do we handle this? This pattern will repeat for dam/jam surveys in regular DCEs.

**Comment by joewheaton** [comment]
@Jdgilby and @nick4rivers just drawing your attention to this. Have we resolved this satisfactorily in "complexes" for design and for dam/jam surveys? @KellyMWhitehead and I discussed briefly and we have an idea for how to address this.

**Comment by guswathen** [comment]
I was about to put in a request for functionality but lo! Here we are. I have a very common practice of making build maps with the 'Structure Complex Extent' as my definer of the extent of the map (usually use the atlas functionality for making multiple maps). I often report the number of LTPBR structures with a complex on the map. In previous versions there was a column in the Extent that summed the number of structures, however, there is no longer. Is there anyway to get this back? Given the long list of potential structure types, I think summing by type is excessive and can be done in some table in excel. However, just number of structures would be really cool. Thank you for considering. - Gus

**Reply by joewheaton** [reply]
Thanks @guswathen for this feedback. I will discuss tomorrow with @KellyMWhitehead and we will see what we can do. We are meant to have this capability, but it might just be lost in the shuffle and needs to be reconnected. Stay tuned.
